src/uct/ib/dc/dc_mlx5.c

```diff
@@ -470,9 +470,10 @@ ucs_status_t uct_dc_mlx5_iface_create_dct(uct_dc_mlx5_iface_t *iface)
     attr.qp_state                  = IBV_QPS_RTR;
     attr.path_mtu                  = iface->super.super.config.path_mtu;
     attr.min_rnr_timer             = iface->super.super.config.min_rnr_timer;
+    attr.ah_attr.is_global         = uct_ib_iface_is_roce(&iface->super.super.super);
```

Comment:
should we use iface->super.super.super.is_global_addr?
verbs dct does not support global address on IB?